hack/clean-leaked.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# network config
+# TODO - this should be configurable, using hardcoded values now
+IPV4_CIDR="169.254.64.0/24"
+IPV6_CIDR="64:ff9b::/96"
+NAT_IF="nat64"
+
+# dev setup config
+KIND_CLUSTER="ipv6"
+CONTAINERD="docker"
+
+# $1 - node
+# rest - args
+function exec_kind_node {
+    ${CONTAINERD} exec -it "$@"
+}
+
+
+for kind_node in `kubectl get nodes | tail -n +2 | awk '{ print $1 }'`; do
+    # fetch iface with default route
+    GWIF="`exec_kind_node ${kind_node} ip route | grep default | awk '{ print $5 }'`"
+
+    # clean iptables rules
```

Comment:
I've added this logic to the golang program, is it not cleaning it?

the program should delete everything on exit